app/javascript/components/analytics/widgets/MonitoreeFlow.js

```diff
@@ -22,70 +22,91 @@ class MonitoreeFlow extends React.Component {
           transferred_in: thisTimeFrameData?.transferred_in || 0,
           closed: thisTimeFrameData?.closed || 0,
           transferred_out: thisTimeFrameData?.transferred_out || 0,
+          exposure_to_isolation: thisTimeFrameData?.exposure_to_isolation || 0,
+          isolation_to_exposure: thisTimeFrameData?.isolation_to_exposure || 0,
         };
       });
     });
   }
 
+  renderWorkflowTable(data, index) {
+    let oppositeWorkflow = _.upperCase(WORKFLOWS[Number(index == 0 ? 1 : 0)]);
+    return (
+      <Col lg="12">
+        <table className="analytics-table">
+          <thead>
+            <tr>
+              <th className="py-0"></th>
+              {MONITOREE_FLOW_HEADERS.map((monitoreeFlowHeader, index) => (
+                <th key={index}>{monitoreeFlowHeader}</th>
+              ))}
+            </tr>
+          </thead>
+          <tbody key={index}>
+            <tr>
+              <td className="font-weight-bold text-left p-0">
+                <u>{_.upperCase(WORKFLOWS[Number(index)])} WORKFLOW</u>
+              </td>
+            </tr>
+            <tr style={{ height: '20px' }}>
```

Comment:
Thoughts on making this 25px and then adding an `align-bottom` class to the td?

Reply:
Definitely like this better https://github.com/SaraAlert/SaraAlert/pull/834/commits/ce3fdacb48fc1507331a644e46a32b5a69c024b6

Reply:
Can we do this on the outgoing one as well?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/834/commits/b6e261c280b239bab2ccf9c4c0bedbf84ab6b745